### DISCUSSION TITLE
Update of Google Coral models

### DISCUSSION THREAD

**Discussion by chokays** [blakeblackshear/frigate#4940] [Ask A Question]
When I set up Frigate, I installed the Google Coral things as well. Do I need to update the Google Coral stuff from time to time? Would this improve object or person detection?

**Comment by NickM-27** [comment]
If you have an m.2 based coral then you may want to look if there is a newer driver available. But there is nothing to update for USB corals, also the runtime is kept up to date in the frigate container. 

> Would this improve object or person detection?

The model is provided by frigate itself, updating the driver won't affect this.
